src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -294,6 +294,7 @@ function test_reuse_osd_id() {
     run_mon $dir a || return 1
     run_mgr $dir x || return 1
     create_rbd_pool
+    ceph osd pool create bar 128
```

Comment:
@liewegas 
ceph-disk has a set of rather convoluted and intrincate ways to find out where its components and modules are. But it only assumes that it is either in a normal installed running environment: /usr/bin or /usr/local/bin, OR in ./bin when running from build.
Otherwise you need to set some vars, like:
```
export PYTHONPATH=./pybind:/usr/srcs/Ceph/work/ceph/src/pybind:/usr/srcs/Ceph/work/ceph/build/lib/cython_modules/lib.2:
export LD_LIBRARY_PATH=/usr/srcs/Ceph/work/ceph/build/lib
```